apps/api/Transcript Summarization.postman_collection.json

```diff
@@ -1,155 +1,246 @@
 {
```

Comment:
Check this query.
```sql
# Write your query or mutation here
{
  summaries(
    options: {
      limit: 10
      offset: 0
      sortOrder: ASC
      filters: [
        { field: "createdOn", operator: "gte", value: "2023-01-01" } #Specify the date range to fetch record
        { field: "createdOn", operator: "lte", value: "2024-06-06" }
        { field: "createdBy", operator: "in", value: "[\"harish@osmosys.co\",\"admin2\"]" } # Specify the email id of the person whose record you want to find 
      ]
    }
  ) {
    limit
    offset
    summaries {
      createdBy
      createdOn
      jobId
      inputFile
      jobStatus
      modifiedBy
      modifiedOn
      outputText
      status
    }
    total
  }
}

```

Reply:
updated in https://github.com/OsmosysSoftware/osm-transcript-summarizer/pull/38/commits/8e9bb841a0c9614e1bc4ebe40eb96e846b54b6cd